src/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+# PC file parser
+find_package(BISON REQUIRED)
+find_package(FLEX REQUIRED)
+BISON_TARGET(PcParser cps/pc_compat/pc.y ${CMAKE_CURRENT_BINARY_DIR}/cps/pc_compat/pc.parser.cpp)
+FLEX_TARGET(PcScanner cps/pc_compat/pc.l ${CMAKE_CURRENT_BINARY_DIR}/cps/pc_compat/pc.lexer.cpp)
+ADD_FLEX_BISON_DEPENDENCY(PcScanner PcParser)
```

Comment:
For what it's worth, CMake is case-insensitive for function and macro names. It's a style concern whether you want to type these accessors as ALL_CAPS or not. I would suggest using lower_case. Additional comments usually do a better job of drawing attention to specific APIs if that's necessary here, especially because comments could explain why instead of just YELLING_CASE.